bindings/c/CONTRIBUTING.md

```diff
@@ -1,33 +1,38 @@
 # Contributing
-- [Contributing](#contributing)
-   - [Setup](#setup)
-      - [Using a dev container environment](#using-a-dev-container-environment)
-      - [Bring your own toolbox](#bring-your-own-toolbox)
-   - [Build](#build)
-   - [Test](#test)
-   - [Docs](#docs)
-   - [Misc](#misc)
+
+1. [Contributing](#contributing)
```

Comment:
we can use `1.` everywhere to enable auto order.

Reply:
I need some changes on my auto-formatting tools, actually I am always using numbered lists 🤣

Reply:
Ok, not a big issue, let's go!